Core/Data.EF/ApplicationDbContext.cs

```diff
@@ -94,6 +97,7 @@ public override int SaveChanges(bool acceptAllChangesOnSuccess)
     public override async Task<int> SaveChangesAsync(bool acceptAllChangesOnSuccess, CancellationToken cancellationToken = default)
     {
         SetCreatedUpdatedFields();
+        InitializeTranslation();
```

Comment:
Do we really want this in the ApplicationDbContext and forcing every context to use this? @jstorteboom